lib/wibox/layout/align.lua

```diff
@@ -164,6 +165,12 @@ function align:set_third(widget)
     self:emit_signal("widget::layout_changed")
 end
 
+--- Get all children of this layout
+-- @treturn table a list of all widgets
+function align:get_children()
+    return {self.first, self.second, self.third}
```

Comment:
I think the `from_sparse` here is still needed. Sorry!
(If `first` and `second` is nil.... oh and the recursive function will do `(nil):get_children()`)